app/kumactl/cmd/install/render_helm_files_no_gateway.go

```diff
@@ -0,0 +1,18 @@
+//go:build !gateway
+// +build !gateway
+
+package install
+
+import (
+	"strings"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+)
+
+func init() {
+	helmTemplateFilters = []fileFilter{
+		func(f data.File) bool {
+			return !strings.Contains(f.FullPath, "crd") || !strings.Contains(f.Name, "gateway")
```

Comment:
So one idea that occured to me is to unmarshal the CRD into a `CustomResourceDefinition`, then to verify whether the kind is in the global type registry. That would automatically filter gateway types out correctly with no special cases. WDYT?